README.md

```diff
@@ -117,6 +117,30 @@ Follow the given instructions for Login into the app.
 
 ## Environment Variables
 
+Use `.env.template` file to make a new `.env` file and add the following details:
+
+```
+SECRET_KEY=<your-secret-key>
+EMAIL_USER=<email-address>
+
+```
+
+This environment variable is optional for development
+```
+SENDGRID_API_KEY=<your-sendgrid-api-key>
+```
+
+Where the database is necessary, you will need to provide the following DB details in the `.env` file
+
+ ```
+DB_BACKEND=postgresql
+DB_NAME=<your-db-name or osp>
+DB_USERNAME=<your-db-username or osp>
+DB_PASSWORD=<your-db-password or osp>
+DB_HOST=localhost
+DB_PORT=5432
+```
```

Comment:
No need to mention about all environment variables as it is defined already in README.

Reply:
Also, I think you haven't removed the above lines.
You don't need to explain all the envs variables.
Use this:
**Use `.env.template` file to make a new `.env` file. You can read about environment variables used in this here(use reference for env-variables section.)**